include/swupd.h

```diff
@@ -19,17 +19,21 @@
 #if SWUPD_WITH_BSDTAR
 #define TAR_COMMAND "bsdtar"
 #define TAR_XATTR_ARGS ""
+#define TAR_XATTR_ARGS_STRLIST
```

Comment:
@bryteise As far as I remember bsdtar manage extended attributes by default, you don't need to specify.